### PR TITLE
Update ai_tracker.sc

### PR DESCRIPTION
**Pull request by Ghoulboy78**
Splitting #618 into separate prs for each app

### CODE REVIEW

File: src/main/resources/assets/carpet/scripts/ai_tracker.sc [hunk 1]
```diff
@@ -145,8 +87,9 @@ global_functions = {
                   [10+half_width,10+height,10+half_width],
                   0x65432100, 'buddy detection', false
             );
+            [x,y,z] = pos(e);
             current_id = e~'id';
-            buddies = entity_area('villager', e, 10, 10, 10);
+            buddies = entity_area('villager', x, y+height/2, z, 10+half_width, 10+height/2, 10+half_width);
```
Comment (gnembon) [inline src/main/resources/assets/carpet/scripts/ai_tracker.sc:92]: any reason why this?

File: src/main/resources/assets/carpet/scripts/ai_tracker.sc [hunk 2]
```diff
@@ -517,8 +494,9 @@ __tick_tracker() ->
       return()
    );
    p = player();
+   [px, py, pz] = pos(p);
    in_dimension(p,
-      for (entity_area('valid', p, global_range, global_range, global_range),
+      for (entity_area('valid', px, py, pz, global_range, global_range, global_range),
```
Comment (gnembon) [inline src/main/resources/assets/carpet/scripts/ai_tracker.sc:499]: again, why?

File: src/main/resources/assets/carpet/scripts/ai_tracker.sc [hunk 3]
```diff
@@ -381,6 +324,57 @@ global_functions = {
...
+__config() ->{
+    'commands'->{
+        ''->'_command',
+        'clear'->'clear',
+        'toggle boxes'->_()->global_display_boxes = !global_display_boxes,
+        'toggle <display>'->['__toggle',null],
+        'toggle villager <villager_display>'->_(d)->__toggle('villager_'+d,null),
+        'toggle villager hostile_detection <hostile>'->_(h)->__toggle('villager_hostile_detection',h),
+        'update_frequency <ticks>'->_(ticks)->(global_interval = ticks;global_duration = ticks + 2),
+        'transparency <alpha>'->_(alpha)->global_opacity = floor(alpha)
```
Comment (gnembon) [inline src/main/resources/assets/carpet/scripts/ai_tracker.sc:351]: if alpha is in, doesn't need to be floored, right?

Comment (Ghoulboy78) [reply src/main/resources/assets/carpet/scripts/ai_tracker.sc:351]: ah ye ofc, just copied old code cos dumdum

File: src/main/resources/assets/carpet/scripts/ai_tracker.sc (hunk 2 above)
Comment (Ghoulboy78) [reply src/main/resources/assets/carpet/scripts/ai_tracker.sc:499]: again, err... idk...

File: src/main/resources/assets/carpet/scripts/ai_tracker.sc (hunk 1 above)
Comment (Ghoulboy78) [reply src/main/resources/assets/carpet/scripts/ai_tracker.sc:92]: err... idk...

**Pull request merged**